### PR TITLE
Added the examples to an example table

### LINKED ISSUE #10055: chrome_settings_overrides improvements
## What page(s) did you find the problem on?

[chrome_settings_overrides](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/chrome_settings_overrides)

## What is the problem?

Page is not well align with other key pages? That is:

-    add an example row into the table (suggest using the homepage example)
-    move the 2 examples to an "Examples" section placed before the "Browser compatibility" section
-    replace the introductory description and 2 bullets with something similar to "Use the chrome_settings_overrides key to override the browser's home page and add a new search engine."

### PR DESCRIPTION
**Pull request by Biki-das**
fixes #10055 

i am kinda confused with the introductory lines @rebloor if you could elaborate i can further push it , rest hope the example table is fine , ping me up anytime for anything

### CODE REVIEW

**Comment by Biki-das** [comment]
@rebloor please review

**Review by rebloor — CHANGES_REQUESTED** [review]
@Biki-das thanks for your help, please let me know if you need any further explanation

**Comment by Biki-das** [comment]
@rebloor tried my best to do as you said check and if you find errors and something to change ping me up!

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/chrome_settings_overrides/index.md [hunk 1]
```diff
@@ -180,6 +169,20 @@ The `chrome_settings_overrides` key is an object that may have the following pro
   </tbody>
 </table>
 
+## Example
+
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/manifest.json/chrome_settings_overrides/index.md:173]: An introduction here would be nice, something such as "This example shows how to set a search provider."

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/chrome_settings_overrides/index.md [hunk 2]
```diff
@@ -180,6 +169,20 @@ The `chrome_settings_overrides` key is an object that may have the following pro
...
+```json
+"chrome_settings_overrides": {
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/manifest.json/chrome_settings_overrides/index.md:175]: Please add the code indenting

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/chrome_settings_overrides/index.md [hunk 3]
```diff
@@ -21,30 +21,19 @@ browser-compat: webextensions.manifest.chrome_settings_overrides
       <th scope="row">Mandatory</th>
       <td>No</td>
     </tr>
+  <tr>
+      <th scope="row">Example</th>
+      <td><pre class="brush:json">
+"chrome_settings_overrides" : {
+"homepage": "https://developer.mozilla.org/"
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/manifest.json/chrome_settings_overrides/index.md:28]: Please add the code indenting

**Review by rebloor — CHANGES_REQUESTED** [review]
Thanks, @Biki-das we are getting there, I've left a few comments

**Comment by Biki-das** [comment]
@rebloor  done with the requested changes

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/chrome_settings_overrides/index.md (hunk 3 above)
Comment (rebloor) [reply files/en-us/mozilla/add-ons/webextensions/manifest.json/chrome_settings_overrides/index.md:28]:
``` suggestion
  "homepage": "https://developer.mozilla.org/"
```

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/chrome_settings_overrides/index.md [hunk 4]
```diff
@@ -180,6 +169,22 @@ The `chrome_settings_overrides` key is an object that may have the following pro
...
 </table>
 
+## Example
+This example shows how to set a search provider.
+
+
+```json
+"chrome_settings_overrides": {
+    "search_provider": {
+    "name": "Discogs",
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/manifest.json/chrome_settings_overrides/index.md:179]:
``` suggestion
      "name": "Discogs",
```

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/chrome_settings_overrides/index.md [hunk 5]
```diff
@@ -180,6 +169,22 @@ The `chrome_settings_overrides` key is an object that may have the following pro
...
+    "search_url": "https://www.discogs.com/search/?q={searchTerms}",
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/manifest.json/chrome_settings_overrides/index.md:180]:
``` suggestion
      "search_url": "https://www.discogs.com/search/?q={searchTerms}",
```

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/chrome_settings_overrides/index.md [hunk 6]
```diff
@@ -180,6 +169,22 @@ The `chrome_settings_overrides` key is an object that may have the following pro
...
+    "keyword": "disc",
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/manifest.json/chrome_settings_overrides/index.md:181]:
``` suggestion
      "keyword": "disc",
```

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/chrome_settings_overrides/index.md [hunk 7]
```diff
@@ -180,6 +169,22 @@ The `chrome_settings_overrides` key is an object that may have the following pro
...
+    "favicon_url": "https://www.discogs.com/favicon.ico"
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/manifest.json/chrome_settings_overrides/index.md:182]:
``` suggestion
      "favicon_url": "https://www.discogs.com/favicon.ico"
```

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/chrome_settings_overrides/index.md [hunk 8]
```diff
@@ -180,6 +169,22 @@ The `chrome_settings_overrides` key is an object that may have the following pro
...
+ }
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/manifest.json/chrome_settings_overrides/index.md:183]:
``` suggestion
  }
```

**Review by rebloor — CHANGES_REQUESTED** [review]
@Biki-das  I've added some suggestions to finish up the code indenting

**Review by rebloor — APPROVED** [review]

**Comment by rebloor** [comment]
Thanks for your help @Biki-das

**Pull request merged**